src/main/java/com/asu/cse545/group12/domain/Transaction.java

```diff
@@ -0,0 +1,154 @@
+package com.asu.cse545.group12.domain;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+public class Transaction {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="TRANSACTIONID")
+	private int transactionId;
+
+	@Column(name = "USERID")
+	private int  userId;
+	
+	@Column(name = "TRANSACTION_STATUS")
+	private String transactionStatus;
+	
+	
+	@Column(name = "CREATION_TIMESTAMP")
+	private Date creationTimestamp;
+	
+	@Column(name = "TRANSACTION_TYPE")
+	private String transactionType;
```

Comment:
Hello,

transaction_kind will say if it is critical or non-critical.
transaction_type will say if it  is debit or credit .


Thanks
Jaswitha

On Thu, Oct 1, 2015 at 10:58 PM, Nagarjuna Myla <notifications@github.com>
wrote:

> In src/main/java/com/asu/cse545/group12/domain/Transaction.java
> <https://github.com/chasethenag420/sbs/pull/13#discussion_r40993201>:
>
> > +	@GeneratedValue(strategy = GenerationType.IDENTITY)
> > +	@Column(name="TRANSACTIONID")
> > +	private int transactionId;
> > +
> > +	@Column(name = "USERID")
> > +	private int  userId;
> > +	
> > +	@Column(name = "TRANSACTION_STATUS")
> > +	private String transactionStatus;
> > +	
> > +	
> > +	@Column(name = "CREATION_TIMESTAMP")
> > +	private Date creationTimestamp;
> > +	
> > +	@Column(name = "TRANSACTION_TYPE")
> > +	private String transactionType;
>
> what is difference between transaction_type and transaction_kind columns?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/13/files#r40993201>.
>